catalog/mapbiomas-public/projects_mapbiomas-public_assets_brazil_lulc_collection9_mapbiomas_collection90_integration_v1.jsonnet

```diff
@@ -0,0 +1,367 @@
+local id = 'projects/mapbiomas-public/assets/brazil/lulc/collection9/mapbiomas_collection90_integration_v1';
+
+local subdir = 'mapbiomas-public';
+
+local version = '1.0';
+
+local ee_const = import 'earthengine_const.libsonnet';
+local ee = import 'earthengine.libsonnet';
+local spdx = import 'spdx.libsonnet';
+local units = import 'units.libsonnet';
+
+local license = spdx.cc_by_4_0;
+
+local basename = std.strReplace(id, '/', '_');
+local self_ee_catalog_url = ee_const.ee_catalog_url + basename;
+
+{
+  id: id,
+  
+  title: 'MapBiomas Land Use and Land Cover - Brazil Collection 9.0 V' + version,
+
+  version: version,
+
+  description: |||
+    MapBiomas Collection 9 is an annual Land Use and Land Cover (LULC) dataset for Brazil, 
```

Comment:
Would you be interested in creating an image collection instead? Then it will be easier for you to update it in the future (you won't have to reupload the whole image), and you'll be able to set dates on each image, allowing users to use the standard filterDate())